hack/cluster-deploy.sh

```diff
@@ -55,6 +55,9 @@ function _ensure_cdi_deployment() {
     override="https://127.0.0.1:$host_port"
     _kubectl patch cdi ${cdi_namespace} --type merge -p '{"spec": {"config": {"uploadProxyURLOverride": "'"$override"'"}}}'
 
+    # Configure overhead for rook-ceph-block fs volumes to be slightly higher than default
+    _kubectl patch cdi ${cdi_namespace} --type merge -p '{"spec": {"config": {"filesystemOverhead": {"global": "0.07"}}}}'
```

Comment:
Just a thought: who is responsible to do this patch in an openshift cluster? is it HCO? if so maybe need to notify them about this?

Reply:
dropped for now. pretty ugly problem, since patching the fs overhead in the tests means they must be serial (patching state)